stdlib/list.mli

```diff
@@ -230,6 +230,15 @@ val find_opt: ('a -> bool) -> 'a list -> 'a option
     satisfies [p] in the list [l].
     @since 4.05 *)
 
+val find_some: ('a -> 'b option) -> 'a list -> 'b option
+(** [find_some f l] applies [f] to every element of [l] in turn,
```

Comment:
```suggestion
(** [find_some f l] applies [f] successively to elements of [l],
```

might remove the wrong impression that the function is indeed applied to all element.